app/_src/quickstart/kubernetes-demo.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Deploy Kuma on Kubernetes
+---
+
+To start learning how {{site.mesh_product_name}} works, you run and secure a simple demo application that consists of two services:
+
+- `demo-app`: a web application that lets you increment a numeric counter. It listens on port 5000
+- `redis`: data store for the counter
+
+## Prerequisites
+- [Helm](https://helm.sh/) - a package manager for Kubernetes
+- [Kind](https://kind.sigs.k8s.io/) - a tool for running local Kubernetes clusters
+
+## Start Kubernetes cluster
+
+Start a new Kubernetes cluster on your local machine by executing:
+
+```sh
+kind create cluster --name=kuma
+```
+
+{% tip %}
+You can skip this step if you already have a Kubernetes cluster running.
+It can be a cluster running locally or in a public cloud like AWS EKS, GCP GKE, etc.
+{% endtip %}
+
+## Install Kuma
+
+Install Kuma control plane with Helm by executing:
+
+```sh
+helm repo add kuma https://kumahq.github.io/charts
+helm repo update
+helm install --create-namespace --namespace {{site.mesh_namespace}} {{ site.mesh_helm_install_name }} {{ site.mesh_helm_repo }}
+```
+
+## Deploy demo application
+
+1.  Deploy the application
+    ```sh
+    kubectl apply -f https://raw.githubusercontent.com/kumahq/kuma-counter-demo/master/demo.yaml
+    ```
+
+2.  Port-forward the service to the namespace on port 5000:
+
+    ```sh
+    kubectl port-forward svc/demo-app -n kuma-demo 5000:5000
+    ```
```

Comment:
Depending on what type of audience you are aiming for here you may want to say something about waiting fot he pods to be running

> error: unable to forward port because pod is not running. Current status=Pending

Kinda semi-obvious from the error log but you never know 🤷 

Completely up to you tho

Reply:
make sense, added wait command